### DISCUSSION TITLE
blog discussion - Hello world

### DISCUSSION THREAD

**Discussion by giscus** [fiqryq/Fiqrychoerudin.dev#82] [Blog]
# blog discussion - Hello world

helo world from description.

https://www.fiqry.dev/blog/post/hello-world

**Comment by fiqryq** [comment]
## what u looking at?

<img src='https://avatars.githubusercontent.com/u/25787603?v=4' width='100%'/>

**Reply by fiqryq** [reply]
handsome boy!!!
